flink-runtime/src/test/java/org/apache/flink/runtime/leaderretrieval/ZooKeeperLeaderRetrievalConnectionHandlingTest.java

```diff
@@ -317,6 +318,8 @@ private void testWithQueueLeaderElectionListener(
                 leaderRetrievalDriver.close();
             }
         }
+
+        Assertions.fail("Provoke a failure to check the ZooKeeper logs.");
```

Comment:
should this be removed?

Reply:
yes, this will be removed. I added it to verify the log generation.